sos/report/plugins/lvm2.py

```diff
@@ -72,12 +72,16 @@ def setup(self):
         vgs_cols = vgs_cols + ',' + 'vg_tags,systemid'
         lvs_cols = ('lv_tags,devices,lv_kernel_read_ahead,lv_read_ahead,'
                     'stripes,stripesize')
+        self.add_cmd_output("lvs -a -o +%s %s" % (lvs_cols, lvm_opts_foreign),
+                            tags="lvs")
+        self.add_cmd_output(
+            "pvs -a -v -o +%s %s" % (pvs_cols, lvm_opts_foreign),
+            tags="pvs")
+        self.add_cmd_output("vgs -v -o +%s %s" % (vgs_cols, lvm_opts_foreign),
+                            tags="vgs")
```

Comment:
Could we make these tags a bit more specific? I realize this would require _also_ updating the insights specs, but I'm hesitant here for the case of future-proofing where if a new collection is added it could be confusing to have a generic "lvs" tag for example not be intuitively linked to either.

Reply:
Hmm let me see if it's possible to change the spec name, it appears to be a sos specific spec and not an insights collection spec name.

Reply:
@TurboTurtle Would lvs_sos, vgs_sos, pvs_sos be a good enough tag or would you like more description in the tag/spec name?

Reply:
Yeah, that's totally fine.

Reply:
Updated the names, and put in a PR to rename the specs in core.